libs/langchain/langchain/vectorstores/pinecone.py

```diff
@@ -302,6 +333,55 @@ def max_marginal_relevance_search(
             embedding, k, fetch_k, lambda_mult, filter, namespace
         )
 
+    @classmethod
+    def get_pinecone(
```

Comment:
imo we should just expose a `get_pinecone_index(cls, index_name, pool_threads) -> pinecone.Index` class method, otherwise this is confusingly similar to init